src/main/java/net/mcreator/ui/modgui/DimensionGUI.java

```diff
@@ -98,7 +100,7 @@ public class DimensionGUI extends ModElementGUI<Dimension> {
 
 	private BiomeListField biomesInDimension;
 
-	private final DataListComboBox igniterTab = new DataListComboBox(mcreator);
+	private final TabListField creativeTabs = new TabListField(mcreator);

```

Comment:
![image](https://github.com/MCreator/MCreator/assets/16374228/82c5a742-a2c6-45a4-a351-3c30794b205c)

This field is not disabled when the portal is not enabled. Once again, please do more thoughtful tests of your PRs, I can't doubt and manually check every single line. I can't accept big PRs if they are untested to this degree...

If you do change around something, you need to test all potentially impacted areas